From f5177e740846c074e0e2f7aae04a19190e6cbda8 Mon Sep 17 00:00:00 2001
From: Neal Gompa <neal@gompa.dev>
Date: Thu, 4 May 2023 13:52:01 -0400
Subject: [PATCH] UI: Default to fdk-aac for AAC when auto-configuring settings

We always have the FDK-AAC codec available, and it's a better
codec anyway.
---
 UI/window-basic-auto-config-test.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/UI/window-basic-auto-config-test.cpp b/UI/window-basic-auto-config-test.cpp
index 5b5051919..9c4769e0a 100644
--- a/UI/window-basic-auto-config-test.cpp
+++ b/UI/window-basic-auto-config-test.cpp
@@ -194,7 +194,7 @@ void AutoConfigTestPage::TestBandwidthThread()
 
 	OBSEncoderAutoRelease vencoder = obs_video_encoder_create((wiz->x264Available ? "obs_x264" : "ffmpeg_openh264"),
 								  "test_h264", nullptr, nullptr);
-	OBSEncoderAutoRelease aencoder = obs_audio_encoder_create("ffmpeg_aac", "test_aac", nullptr, 0, nullptr);
+	OBSEncoderAutoRelease aencoder = obs_audio_encoder_create("libfdk_aac", "test_aac", nullptr, 0, nullptr);
 	OBSServiceAutoRelease service = obs_service_create(serverType, "test_service", nullptr, nullptr);
 
 	/* -----------------------------------*/
@@ -559,7 +559,7 @@ bool AutoConfigTestPage::TestSoftwareEncoding()
 
 	OBSEncoderAutoRelease vencoder = obs_video_encoder_create((wiz->x264Available ? "obs_x264" : "ffmpeg_openh264"),
 								  "test_h264", nullptr, nullptr);
-	OBSEncoderAutoRelease aencoder = obs_audio_encoder_create("ffmpeg_aac", "test_aac", nullptr, 0, nullptr);
+	OBSEncoderAutoRelease aencoder = obs_audio_encoder_create("libfdk_aac", "test_aac", nullptr, 0, nullptr);
 	OBSOutputAutoRelease output = obs_output_create("null_output", "null", nullptr, nullptr);
 
 	/* -----------------------------------*/
-- 
2.46.2

